sample_project/Source/sample_project/Stream Layer/PlaneController.cpp

```diff
@@ -0,0 +1,75 @@
+/* Copyright 2023 Esri
+*
+ * Licensed under the Apache License Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#include "PlaneController.h"
+#include "ArcGISMapsSDK/BlueprintNodes/GameEngine/Geometry/ArcGISSpatialReference.h"
+#include "DeadReckoning.h"
+
+APlaneController::APlaneController()
+{
+	PrimaryActorTick.bCanEverTick = true;
+
+	USceneComponent* rootComponent = CreateDefaultSubobject<USceneComponent>(TEXT("Root"));
+	RootComponent = rootComponent;
+	mesh = CreateDefaultSubobject<UStaticMeshComponent>(TEXT("Plane Mesh"));
+	mesh->SetStaticMesh(planeModel);
+	mesh->SetWorldRotation(FRotator(0.0f, 90.0f, 0.0f));
+	mesh->SetupAttachment(rootComponent);
+	LocationComponent = CreateDefaultSubobject<UArcGISLocationComponent>(TEXT("Location Component"));
+	LocationComponent->SetupAttachment(rootComponent);
+
+	UWidgetBlueprintGeneratedClass* Widget = LoadObject<UWidgetBlueprintGeneratedClass>(nullptr, TEXT("WidgetBlueprint'/Game/SampleViewer/Samples/StreamLayer/UserInterface/wbp_PlaneLabel.wbp_PlaneLabel_c'"));
+	TextComponent = CreateDefaultSubobject<UWidgetComponent>(TEXT("TextComponent"));
+	TextComponent->SetupAttachment(rootComponent);
+	TextComponent->SetWidgetClass(Widget);
+	TextComponent->SetWorldLocation(FVector(0.0f, 0.0f, 500.0f));
+	TextComponent->SetWorldRotation(FRotator(0.0f, 180.0f, 0.0f));
+	TextComponent->SetDrawSize(FVector2d(300.0f, 100.0f));
+}
+
+void APlaneController::PredictPoint(double intervalMilliseconds)
+{
+	auto cGroundSpeedKnots = featureData.attributes.speed;
+	//0.541 m/s = conversion rate for Ground speed Knots to meters per second
```

Comment:
```suggestion
	// 0.51444444444 = conversion rate for Ground speed Knots to meters per second
```

Not sure about the capitalization